src/cozy/react/components/BackButton/index.jsx

```diff
@@ -0,0 +1,18 @@
+/* global cozy */
```

Comment:
This is why we have the v8 of the bar. To be able to use hook in it. (but agreed that we should not do the upgrade right now for this project ;))